packages/storybook-ui-components/stories/Tooltip.stories.tsx

```diff
@@ -12,7 +12,7 @@ export default {
 export const Example = () => {
   const text = (
     <span>
-      reasonble tooltip text,
+      reasonbale tooltip text,
```

Comment:
is this meant to fix the spelling error, because it still needs adjustment

Reply:
Done